README.md

```diff
@@ -42,7 +42,12 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+for(var x = 0, x < foods[0].length){
+  console.log("I want to eat a" + foods[0][x])
+}
+
+//I know I'm missing something in this but can't figure out what.
```

Comment:
You need a `x++` or `x += 1` as the third thing in the `for` loop. Otherwise it'll try to run forever.